tasks/libs/ciproviders/github_api.py

```diff
@@ -378,6 +378,26 @@ def create_release(self, tag, message, draft=True):
             draft=draft,
         )
 
+    def get_codereview_complexity(self, pr_id):
+        pr = self._repository.get_pull(pr_id)
+        criteria = {
+            'easy': {'files': 4, 'lines': 150, 'comments': 1},
+            'hard': {'files': 12, 'lines': 650, 'comments': 9},
```

Comment:
Curious to know where these criterias come from? Is that something you computed with existing PRs or is it an esitmation you made?